packages/docusaurus-plugin-content-blog/src/plugin-content-blog.d.ts

```diff
@@ -73,6 +73,13 @@ yarn workspace v1.22.19image` is a collocated image path, this entry will be the
      * Unknown keys are allowed, so that we can pass custom fields to authors,
      * e.g., `twitter`.
      */
+    socials?: {
```

Comment:
The comment above was not meant to be for the `socials` attribute.

Please create 2 separate comments and reword them so that they keep making sense.

We don't want anymore people to add `author.twitter` for example, since now they are expected to write `author.socials.twitter`